_source/logzio_collections/_prometheus-sources/telegraf-bCache.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Ship bCache metrics via Telegraf
+logo:
+  logofile: bcache-telegraf.png
+  orientation: vertical
+data-source: bCache metrics over Telegraf
+templates: ["docker"]
+contributors:
+  - daniel-tk
+  - nshishkin
+shipping-tags:  
+  - prometheus
+order: 800
+---
+
+
+## Overview
+
+Telegraf is a plug-in driven server agent for collecting and sending metrics and events from databases, systems and IoT sensors.
+
+To send your Prometheus-format bCache metrics to Logz.io, you need to add the **inputs.bcache** and **outputs.http** plug-ins to your Telegraf configuration file.
+
+#### Configuring Telegraf to send your metrics data to Logz.io
+
+<div class="tasklist">
+
+##### Set up Telegraf v1.17 or higher
+
+{% include metric-shipping/telegraf-setup.md %}
+
+##### Add the inputs.bcache plug-in
+
+First you need to configure the input plug-in to enable Telegraf to scrape the bCache data from your hosts. To do this, add the following code to the configuration file:
```

Comment:
we need to add a sentence to install the telegraf on the linux server that we want to collect the bcache statistics from